### ISSUE TITLE
FinalizeFailed error 

### ISSUE THREAD

**Issue by kimimhong** [filecoin-project/lotus#5144]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

Proving is done due to normal sealing work
Failure in Finalize phase.
A brief description of the problem you encountered while proving (sealing) a sector.

Even though there is a file normally in the path
Finalize does not proceed successfully and continues to fail
A fault occurs during windowpost.

Including what commands you ran, and a description of your setup, is very helpful.

**Sectors status**

![image](https://user-images.githubusercontent.com/69128780/101444513-fa121780-3962-11eb-84bc-d22db9357107.png)

**Lotus miner logs**

![image](https://user-images.githubusercontent.com/69128780/101444571-1dd55d80-3963-11eb-9b3b-e3ec7294314e.png)

**Lotus miner diagnostic info**
![image](https://user-images.githubusercontent.com/69128780/101444634-3d6c8600-3963-11eb-943d-303fb444d5b0.png)

It was always the latest version and not even before,
The same phenomenon is happening in version 1.2.2 now.

How can I fix it?

I haven't moved any files.

If I keep getting faults I am penalized. ...

**Comment by Shekelme** [comment]
Have your issue been resolved?

**Comment by Reiers** [comment]
Hi @kimimhong 

Thanks for the report. This is fixed in latest release.
Please upgrade Lotus: https://github.com/filecoin-project/lotus/releases/tag/v1.13.1

Closing ticket.

If you are still experiencing problems, open a new ticket or ask in our Discussions [forum.](https://github.com/filecoin-project/lotus/discussions)
